BoxOffice/Model/MovieInformation.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  MovieInformation.swift
+//  BoxOffice
+//
+//  Created by LeeSeongYeon on 2024/03/11.
+//
+
+import UIKit
+
+final class MovieInformation: LoadDataProtocol {
+    
+    typealias LoadedData = MovieDetail?
+    var loadedData: MovieDetail?
+    
+    let movieCode: String
+    private(set) var movieImage: UIImage? = nil {
+        didSet {
+            delegate?.reloadView()
+        }
+    }
+    
+    weak var delegate: DataDelegate?
+    let networkManager: NetworkManager = NetworkManager(urlSession: URLSession.shared)
+    
+    init(movieCode: String) {
+        self.movieCode = movieCode
+    }
+    
+    func loadData() async throws {
+        guard let request = BoxOfficeAPI.movieDetailInformation(movieCode: movieCode).urlRequest else {
+            throw NetworkError.invalidURL
+        }
+        let data: MovieInformationResult = try await self.networkManager.request(request)
+        let movie = data.movieInformationDetail.movie
+        loadedData = converted(movie)
+        movieImage = try await loadImage()
+    }
+    
+    private func loadImage() async throws -> UIImage? {
+        guard let movieName = loadedData?.movieName,
+            let request = KakaoAPI.image(query: movieName).urlRequest else {
+            throw NetworkError.invalidAPIKey
+        }
+        let imageDocument: MovieImageDocument = try await self.networkManager.request(request)
+        guard let imageUrl = imageDocument.documentResults[safeIndex: 0]?.imageURL,
+              let URL = URL(string: imageUrl) else {
```

Comment:
변수명은 lower camel case로 지어볼까요?
```suggestion
              let url = URL(string: imageUrl) else {
```

Reply:
넵!! 🙏